results.pm

```diff
@@ -439,4 +443,31 @@ sub writelog
 	close($fh);
 }
 
+sub openqa_formatter
```

Comment:
This should probably be called writelog_openqa or something similar.

Reply:
openqa_filter is